src/assets/images/dot_line.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="100%" height="35"  viewBox="800 0 300 200">
```

Comment:
I found many icons and images like link.svg, x.svg and others that are used here, so I made the decision to push dot-line.svg here too. Should I use a dot-line.svg in the assets/svg folder?

Reply:
Yes I think it should be moved to `assets/svg`.

But I meant why did you switch from `border-top` to an svg?

Also, you should probably add ` xmlns="http://www.w3.org/2000/svg"` so browsers know how to render.

Reply:
I switched  from `border-top` to an svg  because `border-top-style: dotted;` is a bug on safari. It changes circles to squares.

Here is the explanation.
- https://stackoverflow.com/questions/62518201/css-dotted-hr-is-set-of-little-square-not-dots-cricles

Thanks for your suggestions. I will fix it.

Reply:
I'm done ✨. Please review again.